StormyHares/BangazonApi
# removed DS Store file

## Ticket Link:
  - No Ticket

## Description:
  - Removed DS_Store from repo

## Steps to Test:

  - git fetch
  -git checkout removeDS_Store
  -do an ls -la . in root folder 
- verify .DS_Store file is not there.  (your computer may create it's own copy, but it's now in the gitignore
